README.md

```diff
@@ -1,2 +1,30 @@
 # go-google-scraper-challenge
-A project for Nimble Internal Certification on Web: Go
+A project for Nimble Go Internal Certification on Web
+
+## Development
+
+### Build development dependencies
+
+  ```sh
+  make build-dependencies
+  ```
+
+### Compile assets files
+
+  ```sh
+  make assets
+  ```
+
+### Run the Go application for development
+
+  ```sh
+  make dev
```

Comment:
Not working for me:

![image](https://user-images.githubusercontent.com/696529/102475378-5fb18280-408c-11eb-953e-c98970f6314c.png)

I also had this issue before with Hoang's project due to an incorrect Beego version. Maybe you can try to upgrade to the latest stable version `2.0.1`.

Reply:
@olivierobert  ✅  updated the version on [edc6587](https://github.com/carryall/go-google-scraper-challenge/pull/16/commits/edc6587115da713c4e5236ea95cf2ab89b0d3f35)

Reply:
I can now run the project

![image](https://user-images.githubusercontent.com/696529/102757222-9268bd00-43a3-11eb-8dd3-52e9fbbd36bf.png)

But it seems like there is an issue with the host as it's blank. So in the console it's `http://:8080` instead of `http://localhost:8080`:

![image](https://user-images.githubusercontent.com/696529/102757172-7f55ed00-43a3-11eb-9117-99205d789f6f.png)